container_api/src/container.rs

```diff
@@ -42,6 +44,16 @@ pub struct Container {
     interface_threads: HashMap<String, InterfaceThreadHandle>,
     dna_loader: DnaLoader,
     signal_tx: Option<SignalSender>,
+    network_ipc_uri: Option<String>,
+    network_child_process: NetShutdown,
+}
+
+impl Drop for Container {
```

Comment:
Is Drop some sort of rust standard? I don't see it imported/defined

Reply:
@Connoropolous yar, `Drop` is pretty important it is part of the memory management lifecycle

Reply:
you can hook into things going out of scope

Reply:
that was what I was guessing. Makes sense, thanks David